lib/Controller/PageController.php

```diff
@@ -145,6 +151,10 @@ public function index(string $token = '', string $callUser = '', string $passwor
 			return $this->guestEnterRoom($token, $password);
 		}
 
+		// Internal files that can be opened by Viewer are available only for
+		// registered user.
```

Comment:
Maybe not a long time anymore, I heard viewer for public shares is being worked on